support/client/lib/vwf/view/blockly.js

```diff
@@ -451,11 +450,13 @@ define( [ "module", "vwf/view", "jquery", "vwf/model/blockly/JS-Interpreter/acor
     function cleanUpdateToolbox( xml ) {
         // negateOverlap is being use to negate some of the effect of the MARGIN variable
         // in Blockly.createDom_ when deleting blocks over the flyout
-        var negateOverlap = 35;
+        // SJF: Overlap not present in Blockly with categories enabled
+
+        // var negateOverlap = 35;
         Blockly.updateToolbox( xml );
```

Comment:
Hrm.  Might be better to remove the cruft, and put in a comment saying something like "If you ever want to change back to the simple UI, look in github back to thus-and-such a date/branch."

On a related note, though... are level designers going to have the option to switch between the two UIs, or do we have to pick one and stick with it?